### DISCUSSION TITLE
How to figure out what the thing is doing when it's doing it

### DISCUSSION THREAD

**Discussion by emailing-files-is-still-better** [emailing-files-is-still-better/Ir-Blinky#21] [General]
Pondering life's great questions. Like why #10 is pinging multiple times. How close do the address actually need to be to register. Can we compare what numbers are calculated by how far between each ping they were? Can you got me with a script to spit out the register value based on time and number of repeats?

**Comment by georgev93** [comment]
> Pondering life's great questions. Like why #10 is pinging multiple times. How close do the address actually need to be to register. Can we compare what numbers are calculated by how far between each ping they were? Can you got me with a script to spit out the register value based on time and number of repeats?

Totally. I can got you that script. I think in a perfect world you would just change a variable that dictated "I want a transmission to start every x seconds." In the meantime, the math I'm doing is:

`[Number of repeats] * ( [Time of transmission] + [Time of delay] ) + [Time to calculate the next address]`.

Ideally, we could eliminate that "Time to calculate the next address" because there's plenty of time to calculate that address while the thing is waiting around for the current address to send!

**Reply by georgev93** [reply]
PR #22 covers this

**Comment by georgev93** [comment]
Oh, I thought discussions would be a nice format because you could reply to individual comments and have a "hierarchy" type discussion where it branches off on different paths but is still in the same place. I guess you can only respond if it's not the title message

**Reply by emailing-files-is-still-better** [reply]
I think it kinda works like that?

**Reply by emailing-files-is-still-better** [reply]
on the pc it gives me the reply to comment function rather than just comment.

**Reply by georgev93** [reply]
I wish you could reply to specific replys. We need to go deeper
